BUILDING.md

```diff
@@ -0,0 +1,332 @@
+# Building
+
+This quick overview will help you get started developing in the Babylon Native
+repository. We support development on Windows and macOS. This overview is intended
```

Comment:
Do we want to also include Linux in for development support?

Reply:
I guess it works now.